pandas/core/dtypes/common.py

```diff
@@ -270,7 +271,8 @@ def is_categorical(arr):
 
     Returns
     -------
-    boolean : Whether or not the array-like is of a Categorical instance.
+    boolean
```

Comment:
@datapythonista we have a lot of inconsistency whether we use boolean or bool here in the codebase. Not sure this is easy to validate.

Reply:
I'm not sure if we validate the `Returns` section, but surely the `Parameters`. Not yet in the CI, but we'll fix all the cases soon, and add it.